src/glang/codegen/type_conversions.py

```diff
@@ -227,8 +225,8 @@ def last_type() -> Type:
     if (
         last_type().storage_kind.is_reference()
         and last_type().storage_kind == dest_type.storage_kind
-        and isinstance(last_array_def, (HeapArrayDefinition, StackArrayDefinition))
-        and isinstance(dest_array_def, (HeapArrayDefinition, StackArrayDefinition))
+        and isinstance(last_array_def, HeapArrayDefinition | StackArrayDefinition)
+        and isinstance(dest_array_def, HeapArrayDefinition | StackArrayDefinition)
```

Comment:
Oh wow, cool syntax

Reply:
Turns out, Python has all the features too